app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = Recipe.all
+  end
+
+  def show
+    @recipe = Recipe.find(params[:id])
+  end
+end
```

Comment:
- [ ] **[REQUIRED]** Good job on implementing the `index` and `show` actions in the RecipesController. To meet the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Recipes%20list%3A,to%20delete%20it.) you need to make a few modifications:

To display recipes created by the **logged-in** user: You need to modify the `index` action to only fetch recipes belonging to the logged-in user.